k8s_controller/templates/ac-job-strict.yaml

```diff
@@ -0,0 +1,146 @@
+          volumeMounts:
+            - mountPath: /app/config.toml
+              name: config
+              subPath: config.toml
+          __active: true
+          resources: {}
+        - env:
+            - name: ACBOT_PORT
+              value: '8081'
+            - name: ACBOT_PROXY_HOST
+              value: 127.0.0.1
+          image: aiarena/arenaclient-bot:latest
+          imagePullPolicy: Always
+          name: bot-controller-1
+          ports:
+            - containerPort: 8081
+              name: 8081tcp
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /health
+              port: 8081
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            limits:
+              cpu: '2'
+            requests:
+              cpu: '1'
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+        - env:
+            - name: ACBOT_PORT
+              value: '8082'
+            - name: ACBOT_PROXY_HOST
+              value: 127.0.0.1
+          image: aiarena/arenaclient-bot:latest
+          imagePullPolicy: Always
+          name: bot-controller-2
+          ports:
+            - containerPort: 8082
+              name: 8082tcp
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /health
+              port: 8082
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          resources:
+            limits:
+              cpu: '2'
+            requests:
+              cpu: '1'
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+        - env:
+            - name: ACSC2_PORT
+              value: '8083'
+            - name: ACSC2_PROXY_HOST
+              value: 127.0.0.1
+            - name: SC2PATH
+              value: /root/StarCraftII
+          image: aiarena/arenaclient-sc2:latest
+          imagePullPolicy: Always
+          name: sc2-controller
+          ports:
+            - containerPort: 8083
+              name: 8083tcp
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /health
+              port: 8083
+              scheme: HTTP
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          resources: {}
+      dnsPolicy: ClusterFirst
+      imagePullSecrets:
+      nodeSelector:
+        {}
+      restartPolicy: Never
+      schedulerName: default-scheduler
+      terminationGracePeriodSeconds: 30
+      volumes:
+        - configMap:
+            defaultMode: 424
```

Comment:
I note these permissions are more open than the standard ac-job template, which used 420.
Just raising this in case it can be reverted to 420.